internal/errors.go

```diff
@@ -44,15 +44,27 @@ func TranslateError(err error, isCloudCluster bool, op ...string) (string, bool)
 }
 
 func TranslateClusterError(err error, operation string) (string, bool) {
-	var urlError *url.Error
-	if errors.As(err, &urlError) && strings.Contains(urlError.Error(), "EOF") {
+	var urlErr *url.Error
+	restAPIErr := `Cannot access Hazelcast REST API.
+REST service is disabled in the configuration by default. It enables you to manage cluster with REST-API calls.
+You should enable it on your CLUSTER MEMBERS to use the cluster commands.
+Check this link to find out more: https://docs.hazelcast.com/hazelcast/latest/maintain-cluster/rest-api#enabling-rest-api`
+	if errors.As(err, &urlErr) && strings.Contains(urlErr.Error(), "EOF") {
 		if operation == ClusterShutdown || operation == ClusterChangeState {
-			return "Cannot access Hazelcast REST API. Is it enabled? If yes, check CLUSTER_WRITE endpoint group is enabled https://docs.hazelcast.com/hazelcast/latest/maintain-cluster/rest-api#using-rest-endpoint-groups\nIf not check this link to find out more: https://docs.hazelcast.com/hazelcast/5.0/maintain-cluster/rest-api#enabling-rest-api", true
+			return `Cannot access Hazelcast REST endpoint.
```

Comment:
How about extracting the error message to a constant and move out of the function?

Reply:
Same as above https://github.com/hazelcast/hazelcast-commandline-client/pull/35/commits/b8ea8c4e30cfbde45e4529f4b2926ae3600dee17